src/main/java/nl/tudelft/ewi/devhub/server/database/entities/User.java

```diff
@@ -116,5 +114,23 @@ public boolean isPasswordMatch(String password) {
 		return password != null && this.password != null
 				&& BCrypt.checkpw(password, this.password);
 	}
+
+	public List<Notification> getNotifications() {
+		List<Notification> notifications = new ArrayList<Notification>();
+		for (NotificationsToUsers notificationsToUsers: notificationsToUsersList) {
+			notifications.add(notificationsToUsers.getNotification());
+		}
+		return notifications;
+	}
+
+	public int unreadNotifications() {
+		int amount = 0;
```

Comment:
This can also be written as `notificationsToUserList.stream().filter(NotificationToUser::isRead).count()`.